phoenix-core/src/main/java/org/apache/phoenix/execute/DelegateHTable.java

```diff
@@ -297,4 +297,28 @@ public boolean checkAndDelete(byte[] row, byte[] family, byte[] qualifier,
 		return delegate.checkAndDelete(row, family, qualifier, compareOp, value, delete);
 	}
 
+    @Override
```

Comment:
These methods required for 1.2.2-SNAPSHOT version so currently not required with older versions. will remove it.